mesonbuild/backend/ninjabackend.py

```diff
@@ -24,16 +29,13 @@
 from ..mesonlib import get_meson_script, get_compiler_for_source
 from .backends import CleanTrees, InstallData
 from ..build import InvalidArguments
-import os, sys, pickle, re
-import subprocess, shutil
-from collections import OrderedDict
 
 if mesonlib.is_windows():
-    quote_char = '"'
+    quote_func = lambda s: '"{}"'.format(s)
```

Comment:
__[flake8]__
* [E731] do not assign a lambda expression, use a def

<!-- Comment by [SideCI](https://sideci.com) -->